src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateFormat = date.split(fromFormat[3]);
+  const newDateFormat = [...dateFormat];
+
+  const lenYear = fromFormat.includes('YYYY');
+  const newLenYear = toFormat.includes('YYYY');
+
+  const indexYear = (lenYear
+    ? fromFormat.indexOf('YYYY') : fromFormat.indexOf('YY'));
+  const newIndexYear = (newLenYear
+    ? toFormat.indexOf('YYYY') : toFormat.indexOf('YY'));
```

Comment:
```suggestion
  const indexYear = lenYear
    ? fromFormat.indexOf('YYYY')
    : fromFormat.indexOf('YY');
  const newIndexYear = newLenYear
    ? toFormat.indexOf('YYYY')
    : toFormat.indexOf('YY');
```
скобочки только путают